marketplace-info/marketplace-info_shacl.ttl

```diff
@@ -0,0 +1,135 @@
+                   sh:path marketplace-info:data ],           
+                [ sh:node marketplace-info:BundleDataShape ;
+                  sh:order 3 ;
+                  sh:path marketplace-info:bundleData ] ;
+    sh:targetClass marketplace-info:MarketplaceInfo .
+
+marketplace-info:DomainShape a sh:NodeShape ;
+    sh:property  [ sh:datatype xsd:string ;
+                  sh:in ("hdmap" "environment-model" "ositrace" "scenario" "surface-model" "service" "other" ) ;
+                  sh:message "Validation of domain type failed!"@en ;
+                  sh:description "Choose domain type."@en ;
+                  sh:name "domain"@en ;
+                  sh:order 0 ;
+                  sh:maxCount 1 ;
+                  sh:minCount 1 ;
+                  sh:path marketplace-info:domain ];
+    sh:targetClass marketplace-info:Domain .
+
+marketplace-info:ContentDataShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:string ;
+                  sh:in ("Documentation" "Metadata" "Validation" "Visualization") ;
+                  sh:message "Validation of type failed!"@en ;
+                  sh:description "Choose type of link."@en ;
+                  sh:name "type"@en ;
+                  sh:order 1 ;
+                  sh:maxCount 1 ;
+                  sh:minCount 1 ;
+                  sh:path marketplace-info:dataType ],
+                [ skos:example "link to content data, screenshot, video, routing, 3d preview"@en ;
+                  sh:description "Reference to content data"@en ;
+                  sh:message "Validation of content failed!"@en ;
+                  sh:name "link"@en ;
+                  sh:node marketplace-info:LinkShape ;
+                  sh:order 2 ;
+                  sh:maxCount 1 ;
+                  sh:minCount 1 ;
+                  sh:path marketplace-info:contentData ] ;
+    sh:targetClass marketplace-info:ContentData .
+
+
+
+marketplace-info:DataShape a sh:NodeShape ;
+    sh:property  [ sh:datatype xsd:string ;
+            sh:description "Contract information in regards to the data exchange component." ;
+            skos:example "contract_zcdkr7kqd47y0w5b4tg91w1etw" ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:name "contractId" ;
+            sh:order 1 ;
+            sh:path marketplace-info:contractId ];
+    sh:targetClass marketplace-info:Data .  
+
+marketplace-info:LinkShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:anyURI ;
+                  sh:message "Validation of link failed!"@en ;
+                  sh:description "Enter a link or DID."@en ;
+                  sh:name "link"@en ;
+                  sh:order 0 ;
+                  sh:maxCount 1 ;
+                  sh:minCount 1 ;
+                  sh:path marketplace-info:link ] ;
+    sh:targetClass marketplace-info:Link .
+
+
+marketplace-info:BundleDataShape a sh:NodeShape ;
+    sh:property [ sh:minCount 1 ;
+                  sh:node marketplace-info:RequiredDataShape ;
```

Comment:
Hier und auch in relatedData fehlt mir die description, name, so wie es davor auch im `GeneralShape` war.
würde auch gerne zumindest einen `minCount 0` sehen, auch wenn der default 0 ist.

Reply:
Ich hab jetzt bei requiredData und relatedData den minCount = 0 gesetzt und den Text der Beschreibung entsprechend angepasst. So kann es requiredData oder relatedData  im bundelShape geben